lua/entities/entity_fishing_rod/cl_init.lua

```diff
@@ -39,20 +39,88 @@ function ENT:HUDPaint()
 	if not IsValid(ply) or (ply and not ply.fishingmod) then return end
 	if ply ~= LocalPlayer() and self:GetHook() and self:GetHook():GetPos():Distance(LocalPlayer():EyePos()) > 1500 then return end
 		
-	local xy = ((self:GetBobber() and self:GetBobber():GetPos() or Vector()) + Vector(0,0,10)):ToScreen()
+	local xy = ((self:GetBobber() and self:GetBobber():GetPos() or Vector()) + Vector(0,0,10)):ToScreen() -- kinda unsure about this Vec'0,0,+10'
+
+	local height_offset 		= 40 			-- kinda unsure about this but ok -- it was here before
+	local marginFromBorder 		= 16 			-- 16 pixels from the top and bottom border of the screen/game window
+	local innerBoxXY 			= 3  			-- padding of the 2 shades of background
+	local bg_heightdepthcatch 	= 0  			-- if a catch or depth exist elongate the box
+	local minwid 				= 54 			-- minimum width of dark background box
+	local nickLenLimit 			= 32 			-- Nick Character Limit
+	local tempNick 				= ply:Nick() 	-- temporary nickname manipulation
+	local cntemp2 				= ""
+	local tempNickEC 			= ""
+
+	if EasyChat then -- i know the code looks REAL NASTY but it works with <c=987654> and <color=255,99,9> tags xd dont hate me
+		-- im not a string.professional so i got to create pron
+		local cntemp1 = string.split(string.replace(tempNick,">","<"),"<") -- this is prongarf

```

Comment:
`string.split` and `string.replace` are not valid functions.

Only important if you are not going to use ec_markup, see below.